packages/cosmic-swingset/src/export-kernel-db.js

```diff
@@ -192,6 +193,9 @@ export const initiateSwingStoreExport = (
     abortIfStopped();
     startedKit.resolve();
     log?.(`Starting DB export at block height ${savedBlockHeight}`);
+    // Let the `started` event be sent before proceeding with blocking processing:
+    // `getArtifactNames` may block the agent for a while.
+    await waitUntilQuiescent();
```

Comment:
When testing this in instagoric, I noticed that the previous change was not sufficient. I suspect an `await null` may be enough here, but turning the event loop is much more robust. This is a somewhat strange layering inversion that I don't see how to better handle.

Reply:
this is to let the `log()` get delivered to the parent process? or at least to let it escape the current (child) process? yeah, the full `waitUntilQuiescent()` seems safe.

Reply:
This is to let the `startedKit.promise` reactions run, which include sending a "started" message over to IPC if using `spawnSwingStoreExport`, which is what happens during state-sync.

It's just a bit weird that the export logic has to be conscious of its consumer, but besides having an active "ack" to proceed, I don't see how to better handle this case.